crates/bevy_pbr/src/meshlet/meshlet_bindings.wgsl

```diff
@@ -7,23 +7,19 @@
 struct PackedMeshletVertex {
     a: vec4<f32>,
     b: vec4<f32>,
-    tangent: vec4<f32>,
 }
 
-// TODO: Octahedral encode normal, remove tangent and derive from UV derivatives
 struct MeshletVertex {
     position: vec3<f32>,
     normal: vec3<f32>,
     uv: vec2<f32>,
-    tangent: vec4<f32>,
```

Comment:
Tangents can be oct + one bit for bitangent (though if removing them completely obviously saves even more space)

Reply:
Mhm I saw that yeah. I'd rather the savings. Every byte counts given how much vertex data I'm going to have, and normal maps should be fairly uncommon when it comes to meshlet meshes anyways.